streaming/java/streaming-api/src/main/java/org/ray/streaming/python/PythonFunction.java

```diff
@@ -50,11 +48,9 @@
 
   private PythonFunction(byte[] function,
                          String moduleName,
```

Comment:
`@Nullable` as they can be. Also it would be better we document what `null` means.